### PR TITLE
sci-libs/libticables2: Version bump to 1.3.5, EAPI bump 5 -> 6

### PR DESCRIPTION
**Pull request by g-braeunlich**
Package-Manager: portage-2.3.0

### CODE REVIEW

File: sci-libs/libticables2/libticables2-1.3.5.ebuild [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+
+DEPEND="${RDEPEND}
+	virtual/pkgconfig
+	nls? ( sys-devel/gettext )"
+
+DOCS=( AUTHORS LOGO NEWS README ChangeLog docs/api.txt )
+
+src_prepare() {
+	default
+	eautoreconf
```
Comment (SoapGentoo) [inline sci-libs/libticables2/libticables2-1.3.5.ebuild]: is there a specific reason why you need to autoreconf? (just asking)

File: sci-libs/libticables2/libticables2-1.3.5.ebuild [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+		$(use_enable usb libusb10)
+}
+
+src_install() {
+	use doc && HTML_DOCS+=( docs/html/. )
+	default
+}
+
+pkg_postinst() {
+	elog "Please read README in /usr/share/doc/${PF}"
```
Comment (SoapGentoo) [inline sci-libs/libticables2/libticables2-1.3.5.ebuild]: `elog "Please read README in ${EROOT%/}/usr/share/doc/${PF}"`

File: sci-libs/libticables2/libticables2-1.3.5.ebuild (hunk 1 above)
Comment (g-braeunlich) [inline sci-libs/libticables2/libticables2-1.3.5.ebuild]: If I remember correctly, in all ti* packages, no configure script is provided.
In any case, the build failed without eautoreconf

Comment (SoapGentoo) [inline sci-libs/libticables2/libticables2-1.3.5.ebuild]: Ok, perfect then

**Pull request closed without merging**